.github/workflows/aws-prod.yml

```diff
@@ -2,64 +2,77 @@ on:
   push:
     branches:
       - master
+      - move
 
-name: Deploy to Production
+name: Deploy to AWS Production EKS
 
 jobs:
   deploy:
     name: Deploy
     runs-on: ubuntu-latest
     env:
-      REGION: eu-central-1
-      SERVICE_NAME: flaps-api-prod
+      REGION: us-east-1
+      CLUSTER_NAME: maker-prod
+      SERVICE_NAME: flap-auctions-api
+      AWS_ECR_NAME: flap-auctions-api
+      ENVIRONMENT_TAG: prod
 
     steps:
     - name: Checkout
       uses: actions/checkout@v3
 
     - name: Configure AWS credentials
-      uses: aws-actions/configure-aws-credentials@v1-node16
+      uses: aws-actions/configure-aws-credentials@v2
       with:
         aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         aws-region: ${{ env.REGION }}
 
-    - name: Login to container registry
+    - name: Login to AWS ECR
       id: login-ecr
       uses: aws-actions/amazon-ecr-login@v1
 
-    - name: Build, tag, and push image to container registry
+    - name: Extract commit hash
+      id: vars
+      shell: bash
+      run: |
+        echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+
+    - name: Build, tag, and push image to ECR
       id: build-image
       env:
+        SHA_TAG: ${{ steps.vars.outputs.sha_short }}
+        LATEST_TAG: latest
         ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
-        IMAGE_TAG: latest
-      run: |
-        # Build a docker container and
-        # push it to ECR so that it can
-        # be deployed to ECS.
-        git submodule update --init --recursive
-        docker build -t $ECR_REGISTRY/$SERVICE_NAME:$IMAGE_TAG .
```

Comment:
It looks like this build required git submodules in the past. I would double check and make sure that the repo checkout actually does get the required submodules listed in .gitmodules.

Reply:
Added https://github.com/makerdao/flap-auctions-api/pull/8/commits/8b39ea45a035562ff05ea025d5d1809204b16f4e